client/src/main/java/org/client/Client.java

```diff
@@ -0,0 +1,32 @@
+package org.client;
+
+import org.client.command.MainCommand;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class Client {
+    public static void main(String[] args) {
+        try {
+            URL url = new URL("http://localhost:8080/process");
```

Comment:
we only need             int exitCode = new CommandLine(new MainCommand()).execute(args); here